src/routes/search/components/SearchModal.js

```diff
@@ -87,13 +87,16 @@ class SearchModal extends React.Component<PropsType, StateType> {
 
     const filteredCategories = this.findCategories(categories)
     return <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
-      <SpaceBetween>
-        <View><CategoryList categories={filteredCategories} query={query} onItemPress={this.onItemPress}
-                            theme={theme} language={language} /></View>
-        <SearchFeedbackBox t={t} query={query} theme={theme} resultsFound={filteredCategories.length !== 0}
-                           sendFeedback={sendFeedback} />
-      </SpaceBetween>
-    </ScrollView>
+        <SpaceBetween>
+          <View style={{ minHeight: 70 }}>
```

Comment:
This should be calculated dynamically (depending on the device) or based on the theme.